fetch/main.js

```diff
@@ -0,0 +1,10 @@
+fetch('https://jsonplaceholder.typicode.com/users', {
+  method: 'GET',
+})
+  .then((response) => response.json())
+  .then((data) => {
+    console.log(data);
+  })
+  .catch((error) => {
+    console.error('Error:', error);
+  });
```

Comment:
It would be best to do these with async/await instead of Promise.then. Trust me, once you get used to async/await, you will never want to use Promise.then again!